build/source/driver/multi_driver.f90

```diff
@@ -147,10 +179,17 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 real(dp),allocatable      :: upArea(:)                      ! area upslope of each HRU
 ! general local variables
 real(dp)                  :: fracHRU                        ! fractional area of a given HRU (-)
+integer(i4b)              :: fileUnit                       ! file unit (output from file_open; a unit not currently used)
+character(LEN=256),allocatable :: dataLines(:)    ! vector of character strings from non-comment lines
+character(LEN=256),allocatable :: chardata(:)     ! vector of character data
+integer(i4b)              :: iWord                          ! loop through words in a string
+logical(lgt)              :: flux_mask(size(flux_meta))     ! mask defining desired flux variables
 real(dp),allocatable      :: zSoilReverseSign(:)            ! height at bottom of each soil layer, negative downwards (m)
 real(dp),dimension(12)    :: greenVegFrac_monthly           ! fraction of green vegetation in each month (0-1)
 real(dp),parameter        :: doubleMissing=-9999._dp        ! missing value
 logical(lgt),parameter    :: overwriteRSMIN=.false.         ! flag to overwrite RSMIN
+real(dp)                  :: notUsed_canopyDepth            ! NOT USED: canopy depth (m)
+real(dp)                  :: notUsed_exposedVAI             ! NOT USED: exposed vegetation area index (m2 m-2)
```

Comment:
Note that we call vegetation phenology here to see if we compute the vegetation flux on the first time step. This is done because of the check in coupled_em if computeVegFlux changes in subsequent time steps (if computeVegFlux changes, then the number of state variables changes, and we need to reoranize the data structures)

I have added these comments to the code.